products/pluto/pluto_build/02_build.sh

```diff
@@ -74,8 +74,6 @@ run_sql_file sql/zoning_specialdistrict.sql
 run_sql_file sql/zoning_limitedheight.sql
 run_sql_file sql/zoning_zonemap.sql
 run_sql_file sql/zoning_parks.sql
-run_sql_file sql/zoning_correctdups.sql
-run_sql_file sql/zoning_correctgaps.sql
```

Comment:
is this change related to the new group logic mentioned in the second bullet point?

Reply:
Yup - this deduplication happens via the `GROUP BY` clause now